### ISSUE TITLE
why News?

### ISSUE THREAD

**Issue by Dananay** [manexim/news#8]
Hi,
I just installed your app on my eOS.
I like to support all eOS related devs, so I purchased yours too.
But I fail to understand how this app is a News app.
To me i only shows eOS updates, no news. Is it possible to add other sources?
Could you please, clarify?
Thank you!
~Dan

**Comment by NathanBnm** [comment]
I agree with @Dananay. If this app is intended to present only elementary OS news, you should choose another app name. This could be something like "eOS News", "eOS Feed", "eNews", "eFeed" or something.

**Comment by meisenzahl** [comment]
Hello everybody,

until now the app only shows news from the elementary blog. I'm currently working on being able to select multiple news sources and add your own.

**Comment by devkornev** [comment]
rss?

**Comment by meisenzahl** [comment]
RSS support ist already implemented:

https://github.com/manexim/news/blob/master/src/Controllers/FeedController.vala#L64
